sdcm/nemesis.py

```diff
@@ -434,7 +438,129 @@ def disrupt_stop_start_scylla_server(self):  # pylint: disable=invalid-name
         self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
         self.target_node.start_scylla_server(verify_up=True, verify_down=False)
 
+    def disrupt_start_stop_major_compaction(self):
+        node = self.cluster.nodes[0]
+        compaction_ops = CompactionOps(self.cluster)
+        timeout = 120
+        trigger_func = partial(compaction_ops.trigger_major_compaction)
+        watch_func = partial(compaction_ops.stop_on_user_compaction_logged,
+                             node=node,
+                             watch_for="User initiated compaction",
+                             timeout=timeout,
+                             stop_func=compaction_ops.stop_major_compaction)
+        self.tester.wait_no_compactions_running()
+        ParallelObject.run_once(partial_funcs_w_args=[trigger_func, watch_func],
+                                timeout=timeout)
+
+    def disrupt_start_stop_scrub_compaction(self):
+        node = self.cluster.nodes[0]
+        compaction_ops = CompactionOps(self.cluster)
+        timeout = 120
+        trigger_func = partial(compaction_ops.trigger_scrub_compaction)
+        watch_func = partial(compaction_ops.stop_on_user_compaction_logged,
+                             node=node,
+                             watch_for="Scrubbing",
+                             timeout=timeout,
+                             stop_func=compaction_ops.stop_scrub_compaction)
+        self.tester.wait_no_compactions_running()
+        ParallelObject.run_once(partial_funcs_w_args=[trigger_func, watch_func],
```

Comment:
Let's use the following API: `ParallelObject.run_callables(list_of_callables=[trigger_func, watch_func])`